code/datums/status_effects/status_effect.dm

```diff
@@ -32,11 +34,17 @@
 		A.attached_effect = src //so the alert can reference us, if it needs to
 		linked_alert = A //so we can reference the alert, if we need to
 	if(duration > 0 || initial(tick_interval) > 0) //don't process if we don't care
-		START_PROCESSING(SSfastprocess, src)
+		if(processing_flags & STATUS_EFFECT_FAST_PROCESS)
+			START_PROCESSING(SSfastprocess, src)
+		else if (processing_flags & STATUS_EFFECT_NORMAL_PROCESS)
```

Comment:
Are you sure you want to use flags here? If it's an enum, either of, then use a switch and normal `==` checks.
In general we use bitflags for possible concurrent systems, but the `else if` here tells this is not one of them.

Reply:
I figure it's there if anyone wants to make a SLOW_PROCESS

Reply:
Oh I see what you mean